### ISSUE TITLE
Vulnerable version of googleapis

### ISSUE THREAD

**Issue by sergioregueira** [serverless/serverless-google-cloudfunctions#175]
The latest released version of this package uses a [vulnerable version](https://www.npmjs.com/advisories/791) of `googleapis` package that causes `npm audit` command to fail.

You should update it regardless it does not affect to the deployed functions to avoid errors evaluating the packages.

```
                       === npm audit security report ===                        
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Improper Authorization                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ googleapis                                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=38.0.0                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ serverless-google-cloudfunctions [dev]                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ serverless-google-cloudfunctions > googleapis                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/791                             │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 high severity vulnerability in ***** scanned packages
  1 vulnerability requires manual review. See the full report for details.
```

Thank you so much.

**Comment by mrdulin** [comment]
same issue.

```bash
 === npm audit security report ===                        
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Improper Authorization                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ googleapis                                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=38.0.0                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ serverless-google-cloudfunctions [dev]                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ serverless-google-cloudfunctions > googleapis                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/791                             │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 high severity vulnerability in 884863 scanned packages
  1 vulnerability requires manual review. See the full report for details.


```

**Comment by pmuens** [comment]
@sergioregueira thanks for bringing this up :+1:

This has been fixed via #176. I'll published an updated version to npm in the upcoming hour(s).
